### DISCUSSION TITLE
How do you represent that a tombstone was destroyed?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [muninn/graves#40] [Q&A] [answered]
I assume you:

- create a tombstone with an id, hasStatus = Serviceable, and a startDate and endDate
- create a tombstone with the same id, hasStatus = Demolished ,and a startDate equal to the demolition date

What I can't workout is how to link them using next-instance and previous_instances [as suggested in the Specification](https://rdf.muninn-project.org/ontologies/graves-en.html#newinversion).

Edit: Just noticed the need for an example is on the [To Do list](https://rdf.muninn-project.org/ontologies/graves-en.html#TODO) in the spec 🤪

**Comment by rwarren2** [comment, marked as answer]
This is because of an annoying corner case: some people want to record events that occur on monuments and others just want to records monuments as they are. A typical example is the [ShipWrecks](http://rdf.muninn-project.org/ontologies/graves#ShipWreck) vs the [Ships](https://rdf.muninn-project.org/ontologies/naval-en.html#term_Ship) or [Hull](https://rdf.muninn-project.org/ontologies/naval-en.html#term_Hull)s: it's the same things but it's status has changed along with its position. The other is the reinterment of remains as with the [Canadian War Memorial](https://rdf.muninn-project.org/ontologies/graves-en.html#example_canadian_memorial) example with currently uses the [movedFrom](https://rdf.muninn-project.org/ontologies/graves-en.html#term_movedFrom) instance (See #32).


To solve your immediate problem, if you just want to say the Tombstone only existed from 1861 to 1870.
```
  <graves:Tombstone rdf:about="#TheSpecificTombstone">
   <graves:inscription>A Canadian Soldier of the Great War</graves:inscription>
   <graves:epitaph>Known unto God</graves:epitaph>
   <prov:startedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#gYear">1861</prov:startedAtTime>
   <prov:endedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#gYear">1870</prov:endedAtTime>
  <graves:hasStatus rdf:resources="https://rdf.muninn-project.org/ontologies/graves#Serviceable"/>
  </graves:Tombstone> 
```

if the Tombstone was only ruined and not destroyed in 1807:

```
  <graves:Tombstone rdf:about="#TheSpecificTombstoneRuined">
   <time:after rdf:resource=""/>
   <graves:inscription>A Canadian Soldier of the Great War</graves:inscription>
   <graves:epitaph>Known unto God</graves:epitaph>
   <prov:startedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#gYear">1870</prov:startedAtTime>
  <graves:hasStatus rdf:resources="https://rdf.muninn-project.org/ontologies/graves#Ruins"/>
  </graves:Tombstone> 
```

If you'd like to add a specific event that leads to its demise:
```
 <prov:Activity rdf:about="...">
  <time:after rdf:resource="#TheSpecificTombstone"/>
  <time:before rdf:resource="#TheSpecificTombstoneRuined"/>
  <rdfs:label xml:lang="en">Tombstone destroyed by vandals in 1870</rdfs:label>
  <prov:startedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#gYear">1870</prov:startedAtTime>
  <prov:endedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#gYear">1870</prov:endedAtTime>
 <prov:Activity>
```

Please ignore next_instance and previous_instances from now on.
